pkg/deploy/defaults.go

```diff
@@ -30,21 +30,23 @@ const (
 	DefaultPvcClaimSize                 = "1Gi"
 	DefaultIngressStrategy              = "multi-host"
 	DefaultIngressClass                 = "nginx"
-	DefaultPluginRegistryImage          = "quay.io/eclipse/che-plugin-registry:7.0.0"
+	defaultPluginRegistryImage          = "registry.redhat.io/codeready-workspaces/che-plugin-registry:2.0"
+	defaultPluginRegistryUpstreamImage  = "quay.io/eclipse/che-plugin-registry:7.0.0"
 	DefaultPluginRegistryMemoryLimit    = "256Mi"
 	DefaultPluginRegistryMemoryRequest  = "16Mi"
-	DefaultDevfileRegistryImage         = "quay.io/eclipse/che-devfile-registry:7.0.0"
+	defaultDevfileRegistryImage         = "registry.redhat.io/codeready-workspaces/che-devfile-registry:2.0"
```

Comment:
probably won't be called "che-" anything ... more likely.. 

`	defaultDevfileRegistryImage         = "registry.redhat.io/codeready-workspaces/devfile-registry:2.0"`